database/migrations/2020_06_23_00522_alter_availability_perc_column.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AlterAvailabilityPercColumn extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('availability', function (Blueprint $table) {
+            $table->float('availability_perc', 9, 6)->default(0.000000)->change();
+        });
+    }
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('availability', function (Blueprint $table) {
+            $table->double('availability_perc', 6, 6)->default(0.000000)->change();
```

Comment:
missed the double here.

Reply:
double on all sides, thought i was there ... ;)

Reply:
it should be float in the migration, which translates to double for mysql.

Reply:
ok changed it to float on up and down ... and keep double in schema ...